test/release/examples/primers/modules.chpl

```diff
@@ -0,0 +1,386 @@
+         definitions.
+
+         More details on when overloading applies, when functions may shadow
+         other functions, etc. can be found in the relevant section of the
+         language specification.  They will not be covered further in this
+         primer.
+      */
+
+      use modToUse, DifferentArguments;
+
+      writeln(baz(2, 3));
+      // Accesses the function modToUse.baz using the two arguments.  Should
+      // output 2 * (2 + 3) or '10'
+      writeln(baz(3));
+      // Access the function DifferentArguments.baz using the single argument.
+      // Should output 3 - 2, or '1'
+    }
+
+    writeln();
+    writeln("Limiting a use");
+
+
+    {
+      /* To get around such conflicts, there are multiple strategies.  If only a
+         small number of symbols are desired from a particular module, you can
+         specify the symbols to bring in via an 'only' list.
+
+         Here, because of the 'only' clause in the 'use' of Conflict, Conflict's
+         'bar' is not directly accessible here.
+      */
+      use modToUse;
+      use Conflict only other, another;
+
+      writeln(foo); // Outputs modToUse.foo ('12')
+      writeln(bar); // Outputs modToUse.bar ('2')
+      writeln(other); // Outputs Conflict.other ('5.0 + 3.0i')
+    }
+
+
+    {
+      /* If every symbol other than the one which causes a conflict is desired,
+         you can specify the symbols to exclude via an 'except' list.
+      */
+      use Conflict;
+      use modToUse except bar;
+
+      writeln(foo); // Outputs modToUse.foo ('12')
+      writeln(bar); // Outputs Conflict.bar ('5')
+      writeln(other); // Outputs Conflict.other ('5.0 + 3.0i')
+    }
+
+
+    {
+      /* If both symbols which conflict are desired, or if the use causes
+         symbols to be shadowed which are necessary, you can choose to rename a
+         symbol when including it via the 'as' keyword, so long as the new name
+         does not cause any conflicts with other included symbols.
+      */
+      use modToUse;
+      use Conflict only bar as boop;
+      writeln(bar); // Outputs modToUse.bar ('2')
+      writeln(boop); // Outputs Conflict.bar ('5')
+    }
+
+    writeln();
+    writeln("Application to enums");
+
+    {
+      /* 'Use' statements can also be called on enums.  Normally to access one
+         of an enum's constants, you must provide a prefix of the enum name.
+         With a 'use' of that enum, such a prefix is no longer necessary.
+      */
+
+      enum color {red, blue, yellow};
+
+      // Normally you must use the enum name as a prefix
+      var aColor = color.blue;
+      writeln(aColor);
+
+      use color;
+
+      // The use statement allows you to access an enum's symbols without the
+      // prefix
+      var anotherColor = yellow; // color.yellow
+      writeln(anotherColor);
+
+      // All of the above rules for 'use' statements also apply to 'use's of
```

Comment:
'use' statements -> 'use's of modules

It's not clear to me when you are and are not quoting 'use' -- I'm not sure I care much since it generally works pretty well either as English ("use statement") or a reference to a keyword ("'use' statement), but I wanted to point it out in case you were interested in unifying (or maybe there is a unifying principle I'm not seeing).